TFT/src/User/API/Temperature.c

```diff
@@ -210,6 +214,14 @@ void loopCheckHeater(void)
       heat_update_waiting = true;
     } while (0);
   }
+  else // check temperature auto-report timout and resend M155 command
+  {
+    if (OS_GetTimeMs() > AUTOREPORT_TIMEOUT && !heat_update_waiting)
+    {
+      heat_update_waiting = storeCmd("M155 ");
+      updateNextHeatCheckTime();  // set next timeout for temperature auto-report
```

Comment:
I suggest:
`if (storeCmd("M155 ")`
`{`
`__heat_update_waiting = true;`
`__updateNextCheckTime();`
`}`

or
`if (heat_update_waiting) updateNextHeatCheckTime();`

So if stroeCmd() is unsuccesfull it will not wait another 3 seconds to try to resend "M155 ".